.github/workflows/image-builder-golang.yml

```diff
@@ -0,0 +1,64 @@
+name: builder_image_golang
+on:
+  push:
+    branches: [develop]
+    paths: [images/builder-golang/**, .github/workflows/image-builder-golang.yml]
+  pull_request:
+    paths: [images/builder-golang/**, .github/workflows/image-builder-golang.yml]
+permissions:
+  contents: read
+  pull-requests: read
+  packages: write
+env:
+  GITHUB_REGISTRY: ghcr.io
+  BUILDER_IMAGE_NAME: dyrector-io/dyrectorio/builder-images/golang
+  VERSION: 1
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Add requirements and build the image
+        run: |
+          sudo apt update
+          sudo apt install zstd golang docker.io containerd runc
+          docker build -f ./images/builder-golang/Dockerfile -t ${GITHUB_REGISTRY}/${BUILDER_IMAGE_NAME}:${VERSION} .
```

Comment:
The name already speaks two steps (and three is done: setup, build, export). It would be better be split, no?

Reply:
Right. Fixed.